src/pf-utils/pf-utils.js

```diff
@@ -38,4 +38,76 @@ let I18nUtil = function() {
     self.mixin = mixin;
   };
 };
-export let i18n = new I18nUtil();
+let i18n = new I18nUtil();
+export {i18n};
+
+/**
+ * --------------------------------------------------------------------------
+ * PfUtil
+ * Native Javascript for Bootstrap 3 | Internal Utility Functions
+ * --------------------------------------------------------------------------
+ */
+
+class PfUtil {
+
+  constructor () {
+    this.isIE = (new RegExp("MSIE ([0-9]{1,}[\.0-9]{0,})").exec(navigator.userAgent) !== null) ? parseFloat( RegExp.$1 ) : false;
+    this.mouseHover = ('onmouseleave' in document) ? [ 'mouseenter', 'mouseleave'] : [ 'mouseover', 'mouseout' ];
+    this.tipPositions = /\b(top|bottom|left|top)+/;
```

Comment:
A mouseHover variable doesn't seem generic enough for a util object that could potentially be used by other components. This seems like something this belongs in the component istelf?

Likewise, tipPositions seems very specific to the tooltip component. I'm don't know that this would be useful to other components?